be/src/connector/lake_connector.cpp

```diff
@@ -693,7 +702,6 @@ void LakeDataSource::update_counter() {
     COUNTER_UPDATE(_io_ns_local_disk_timer, _reader->stats().io_ns_local_disk);
     COUNTER_UPDATE(_io_ns_remote_timer, _reader->stats().io_ns_remote);
     COUNTER_UPDATE(_io_ns_total_timer, _reader->stats().io_ns);
-    COUNTER_UPDATE(_io_statistics_timer, _reader->stats().io_ns);
```

Comment:
don't update `_io_statistics`

Reply:
Didn't get what you mean. Could you be more specific?

Reply:
'_io_statistics_timer' this metric is only to offer hierarchy, and this same time can be represents by `_io_ns_total_timer `, so I remove it here, and change its type from `TIME` to `NONE`